src/Components/Home/Home.jsx

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+import img1 from '../../Images/img1.png'
+import img2 from '../../Images/img2.png'
+import img3 from '../../Images/img3.png'
+import img4 from '../../Images/img4.png'
+import img5 from '../../Images/img5.jpg'
+import img6 from '../../Images/img6.jpg'
+import img7 from '../../Images/img7.jpg'
+import img8 from '../../Images/img8.jpg'
```

Comment:
You can import all images in one file and use that file to import any images in one line

Reply:
Yes Pratik I will try this and let you know

Reply:
thanks for the suggestion